spec/after_commit_action_spec.rb

```diff
@@ -18,4 +19,21 @@
     t.array.should include('before_update')
     t.array.should include('after_update')
   end
+
+  context 'when block is executed in another model transaction' do
+    let(:tester)        { Tester.create count: 0 }
+    let(:another_model) { AnotherModel.new tester: tester }
+
+    subject { another_model.save! }
+
+    it { expect { subject }.to change { tester.reload.count }.from(0).to 1 }
+  end
+
+  context 'when there is no transaction' do
```

Comment:
Ahh, yeah, that makes sense, maybe we can just rename that method? `increment_test_counter` or something maybe.